### PR TITLE
feat (#119): gravatar for settings avatar

### PR DESCRIPTION
**Pull request by sliderzxc**

### CODE REVIEW

File: data/src/main/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSource.kt [hunk 1]
```diff
@@ -3,16 +3,20 @@ package io.timemates.backend.data.users.datasource
 import io.timemates.backend.exposed.suspendedTransaction
 import org.jetbrains.annotations.TestOnly
 import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.mod
 import org.jetbrains.exposed.sql.transactions.transaction
 
-class PostgresqlUsersDataSource(private val database: Database) {
+class PostgresqlUsersDataSource(
+    private val database: Database)
```
Comment (y9vad9) [inline data/src/main/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSource.kt:10]: fix formatting

File: data/src/main/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSource.kt [hunk 2]
```diff
@@ -81,13 +95,13 @@ class PostgresqlUsersDataSource(private val database: Database) {
             get(UsersTable.USER_NAME),
             get(UsersTable.USER_EMAIL),
             get(UsersTable.USER_SHORT_DESC),
-            get(UsersTable.AVATAR_FILE_ID)
+            get(UsersTable.AVATAR_FILE_ID),
+            get(UsersTable.GRAVATAR_ID)
```
Comment (y9vad9) [inline data/src/main/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSource.kt:99]: add trailing comma; you can enable it in editor settings and reformat files you changed.

File: data/src/test/kotlin/io/timemates/backend/data/users/UsersRepositoryTest.kt [hunk 3]
```diff
@@ -123,6 +128,7 @@ class UsersRepositoryTest {
             userEmail = "user4@example.com",
             userShortDesc = "4",
             userAvatarFileId = null,
+            gravatarId = null
```
Comment (y9vad9) [inline data/src/test/kotlin/io/timemates/backend/data/users/UsersRepositoryTest.kt:131]: trailing comma

File: data/src/test/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSourceTest.kt [hunk 4]
```diff
@@ -62,7 +63,8 @@ class PostgresqlUsersDataSourceTest {
             userName = testUserName,
             userEmail = email,
             userShortDesc = testBio,
-            userAvatarFileId = null
+            userAvatarFileId = null,
+            gravatarId = null
```
Comment (y9vad9) [inline data/src/test/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSourceTest.kt:67]: trailing comma

File: data/src/test/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSourceTest.kt [hunk 5]
```diff
@@ -87,14 +89,16 @@ class PostgresqlUsersDataSourceTest {
                 userName = "Test User 1",
                 userEmail = email,
                 userShortDesc = "Test Bio 1",
-                userAvatarFileId = null
+                userAvatarFileId = null,
+                gravatarId = null
```
Comment (y9vad9) [inline data/src/test/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSourceTest.kt:93]: trailing comma

File: data/src/test/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSourceTest.kt [hunk 6]
```diff
@@ -87,14 +89,16 @@ class PostgresqlUsersDataSourceTest {
...
             ),
             userId2 to PostgresqlUsersDataSource.User(
                 id = userId2,
                 userName = "Test User 2",
                 userEmail = "user5@example.com",
                 userShortDesc = "Test Bio 2",
-                userAvatarFileId = null
+                userAvatarFileId = null,
+                gravatarId = null
```
Comment (y9vad9) [inline data/src/test/kotlin/io/timemates/backend/data/users/datasource/PostgresqlUsersDataSourceTest.kt:101]: trailing comma

File: settings.gradle.kts [hunk 7]
```diff
@@ -33,12 +33,11 @@ include(
     ":common:page-token",
     ":common:smtp-mailer",
     ":common:cli-arguments",
+    ":common:hashing"
```
Comment (y9vad9) [inline settings.gradle.kts:36]: trailing comma

**Review by y9vad9 — COMMENTED** [review]
overall good job, only thing is missing – it's trailing commas. We use them to avoid history clash (when new parameter is added, line written by you will be marked as line of the other collaborator)

**Review by y9vad9 — APPROVED** [review]

**Review by y9slavabogu — APPROVED** [review]
ya veryu Vadimu

**Pull request merged**